pkg/auth/common/login_handler.go

```diff
@@ -165,6 +178,54 @@ func (h *TanzuLoginHandler) getTokenWithRefreshToken() (*Token, error) {
 	}, nil
 }
 
+// Create or update the cert map entry for the issuer if necessary
+func (h *TanzuLoginHandler) updateCertMap() {
+	// explicitly provided cert info was successfully used to authenticate, so as a
+	// best-effort: save them in the cert map for convenience if necessary
+	if h.tlsSkipVerify || h.caCertData != "" {
+		u, err := url.Parse(h.issuer)
+		if err != nil {
+			fmt.Printf("Unable to parse issuer %s: %v\n", h.issuer, err)
+			return
+		}
+		host := u.Hostname()
+		if host == "" {
+			host = h.issuer
+		}
+
+		var cert *configtypes.Cert
+		found, _ := config.GetCert(host)
+
+		tlsSkipVerifyStr := strconv.FormatBool(h.tlsSkipVerify)
+		if found != nil {
+			if found.CACertData != h.caCertData || found.SkipCertVerify != tlsSkipVerifyStr {
+				cert = &configtypes.Cert{
+					Host:           host,
+					CACertData:     h.caCertData,
+					SkipCertVerify: tlsSkipVerifyStr,
+					Insecure:       found.Insecure,
+				}
+				if err = config.DeleteCert(host); err != nil {
+					log.V(7).Infof("Unable to update cert info: %v\n", err)
+				}
+				err = config.SetCert(cert)
+				if err != nil {
+					log.V(7).Infof("Unable to update cert info: %v\n", err)
```

Comment:
Should we force fail here? Because if we continue it might break plugins later on.

Reply:
I am thinking of leaving as it. The config cert commands are available as a fall back should user need to work around what are some really corner cases.  I will update to always log, however.